### PR TITLE
FeatureEditBaby

### PR DESCRIPTION
**Pull request by Marc120985**

### CODE REVIEW

File: frontend/src/BabyEditPage.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  padding-right: 5rem;
+  background-color: transparent;
+  border: 1px solid var(--color-background);
+  border-radius: 1rem;
+  margin: 0.5rem;
+  color: var(--color-background);
+  font-family: ubuntu, sans-serif;
+`
+const StyledButton3 = styled.button`
+  padding-top: 1rem;
```
Comment (chris-yooo) [inline frontend/src/BabyEditPage.tsx:142]: we need here changes in all files

padding: UP RIGHT DOWN LEFT

File: frontend/src/BabyEditPage.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  text-shadow: 1px 1px 1px var(--color-background);
+  padding: 5px 20px 5px 20px;
+  border: 1px solid var(--color-background);
+  border-radius: 1rem;
+  margin: 1.5rem;
+`
+const StyledSection2 = styled.section`
+  display: flex;
+  justify-content: center;
+  background-color: white;
```
Comment (chris-yooo) [inline frontend/src/BabyEditPage.tsx:114]: use var()

File: frontend/src/BabyEditPage.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+`
+const StyledForm = styled.form`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  background-color: white;
+  align-items: center;
+`
+const StyledLink2 = styled(Link)`
+  padding-top: 1rem;
```
Comment (chris-yooo) [inline frontend/src/BabyEditPage.tsx:81]: same here

File: frontend/src/BabyEditPage.tsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+  padding-right: 5rem;
+  background-color: var(--color-background);
+  display: flex;
+  justify-content: center;
+  border-radius: 1rem;
+  color: var(--color-white);
+  font-family: Ubuntu, sans-serif;
+  text-decoration: none;
+  margin: 0.5rem;
+  border: 1px solid white;
```
Comment (chris-yooo) [inline frontend/src/BabyEditPage.tsx:93]: use var

File: frontend/src/BabyEditPage.tsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    }
+
+
+    return <>
+        <StyledHeader>
+            <h1>{foundBaby.name}</h1>
+        </StyledHeader>
+        <StyledForm>
+            <StyledLabel htmlFor="name">Geburtstag</StyledLabel>
+            <StyledP id="name" value={foundBaby.birthday}></StyledP>
```
Comment (chris-yooo) [inline frontend/src/BabyEditPage.tsx:46]: we didnt forget here the changing of label to input field name

File: frontend/src/App.tsx [hunk 6]
```diff
@@ -35,6 +36,7 @@ export default function App() {
                 <Route path={"/babyoverview"} element={<Babyoverview babies={babies}/>}/>
                 <Route path={"/newbaby"} element={<NewBaby getAllBabies={getAllBabies}/>}/>
                 <Route path={"/baby/:id"} element={<BabyPage babies={babies} getAllBabies={getAllBabies}/>}/>
+                <Route path={"/babyedit/:id"} element={<BabyEditPage babies={babies} getAllBabies={getAllBabies}/>}/>
```
Comment (chris-yooo) [inline frontend/src/App.tsx:39]: ty to react that bugged here 2 hours for s***

**Review by dennis-aclue — APPROVED** [review]
Good Code

**Pull request merged**